src/core/middleware.py

```diff
@@ -49,22 +49,29 @@ def __init__(self, get_response):
         self.get_response = get_response
 
     def __call__(self, request):
-        public_headers = {
+        public_cache_values = {
             'public': True,
             'max-age': settings.CACHE_HEADERS_TTL,
             'stale-while-revalidate': settings.CACHE_HEADERS_TTL,
             'stale-if-error': (60 * 60) * 24, # 1 day, 86400 seconds
         }
-        private_headers = {
+        private_cache_values = {
             'private': True,
             'max-age': 0, # seconds
             'must-revalidate': True,
         }
+        error_cache_values = {
+            'must-revalidate': True,
+            'no-cache': True,
```

Comment:
Following https://stackoverflow.com/a/2068407, `no-cache` is redundant (but harmless).